core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -54,17 +54,20 @@ object ConsumerPerformance {
     val totalBytesRead = new AtomicLong(0)
     val consumerTimeout = new AtomicBoolean(false)
     var metrics: mutable.Map[MetricName, _ <: Metric] = null
+    val joinGroupTimeInMs = new AtomicLong(0)
+    val fetchTimeInMs = new AtomicLong(0)
```

Comment:
There doesn't appear to be any reason for `fetchTimeInMs` to be an `AtomicLong`. We can use a regular long and just initialize it below.